packages/clerk-js/src/core/resources/DisplayConfig.ts

```diff
@@ -10,6 +10,7 @@ export class DisplayConfig extends BaseResource implements DisplayConfigResource
   afterSignOutUrl!: string;
   afterSignUpUrl!: string;
   afterSwitchSessionUrl!: string;
+  isOpen!: boolean;
```

Comment:
❓ I'm unsure why we'd need `isOpen` on `DisplayConfig`. Can you please elaborate?

Reply:
@nikosdouvlis I think you are right on this one since the fields in `DisplayConfig` are mainly static and affected by API calls(ex. in instance)